src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -1132,7 +1135,8 @@
 
 helps['aks nodepool update'] = """
     type: command
-    short-summary: Update a node pool to enable/disable cluster-autoscaler or change min-count or max-count
+    short-summary: Update a node pool properties.
+    long-summary: Update a node pool to enable/disable cluster-autoscaler or change min-count or max-count.  No arguments attempts to move the cluster to its goal state without changing the current cluster configuration. This operation can be used to move out of a non succeeded state.
```

Comment:
```suggestion
    long-summary: Update a node pool to enable/disable cluster-autoscaler or change min-count or max-count.  When called with no optional arguments this attempts to move the cluster to its goal state without changing the current cluster configuration. This can be used to move out of a non succeeded state.
```